app/views/layouts/_logo.html.erb

```diff
@@ -1,6 +1,13 @@
 <a href="/" class="site-logo" aria-label="<%= t("views.main.aria_home") %>">
-  <% if Settings::General.logo_svg.present? %>
-    <%= logo_svg %>
+
+  <% if Settings::General.logo_svg %>
+      <%= Settings::General.logo_svg %>
+  <% elsif FeatureFlag.enabled?(:creator_onboarding) %>
+    <picture style="object-fit: contain;max-width: 300px;max-height: 40px;">
+        <!-- make this dynamic -->
+      <source media="(max-width:768px)" srcset="<%= Settings::General.resized_mobile_logo %>" alt="<%= community_name %>">
+      <img src="<%= Settings::General.resized_web_logo %>" alt="<%= community_name %>">
+    </picture>
```

Comment:
@nickytonline as a last commit to this PR before we send it out for review can you please tackle the following: 
1. I know we have a `$breakpoint-m` defined at `768px` but I didn't know how to reference this var, if it is a var? We will use the breakpoint for `source media="(max-width:768px)"`
2. The SVG is not showing up without a width and height set. If you upload an svg, you'll see it in the DOM but the width is 0px hence we do not see it. 
3. Can you double check if the srcset is working as expected pleas?

Reply:
@nickytonline these are not necessary to address any longer as I changed things around.